README.md

```diff
@@ -21,6 +21,7 @@ Currently, rust-bio provides
 * BWT and FM-Index,
 * FMD-Index for finding supermaximal exact matches,
 * a q-gram index,
+* an orf research algorithm

```

Comment:
Yes, my bad =="
Le 29 mars 2016 15:31, "Johannes Köster" <notifications@github.com> a
écrit :

> In README.md
> <https://github.com/rust-bio/rust-bio/pull/48#discussion_r57722639>:
>
> > @@ -21,6 +21,7 @@ Currently, rust-bio provides
> >  * BWT and FM-Index,
> >  * FMD-Index for finding supermaximal exact matches,
> >  * a q-gram index,
> > +* an orf research algorithm
>
> Do you mean "ORF search algorithm"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-bio/rust-bio/pull/48/files/5ccc54036a28e80c7729883d00d2431991da01c7#r57722639>
>